README.md

```diff
@@ -124,7 +124,7 @@ To run the project, execute the following command:
 
 To run tests, run the following command:
 
-- `bin/rails test test/models/article_test.rb`
+- `bin/rails rspec spec <file_name>.`
```

Comment:
- [ ] As mentioned by the previous reviewer, please update the command to run the tests. It is still incorrect.

Reply:
Fixed.